### ISSUE TITLE
Does it work with 0.16.x?

### ISSUE THREAD

**Issue by danieltomasz** [efemkay/obsidian-modular-css-layout#20]
I installed and enabled the CSS snippets but I dont see any change on 0.16/x Obsidian branch :(

**Comment by efemkay** [comment]
Yeah im not too fluent on creating branch at this moment. So I'm holding out updates to 0.16.x for now. Likely will just release it before end of this month. Just need to figure out how to do beta branch.

**Comment by danieltomasz** [comment]
Thank you for the work! For me having just creating a beta branch without releasing it ( beta  releasing for css  snippets might be tricker than fora  plugin) might  be enough, I might just manually copy css ;)  But I might  wait anyway until public release :)

**Comment by efemkay** [comment]
@danieltomasz i've uploaded the snippets for obsidian 0.16.x support in beta branch -- follow this link below. you may have to manually download the files, still not sure how to properly create release for beta.
- https://github.com/efemkay/obsidian-modular-css-layout/tree/beta-0.16.x
will keep this issue open until obsidian 0.16.x is public

**Comment by danieltomasz** [comment]
Great! When I am using the beta branch and wide-view  enabled only for dataview, it affects the  embedded-images too
